test/e2e/pvutil.go

```diff
@@ -62,19 +62,6 @@ type persistentVolumeConfig struct {
 	namePrefix string
 }
 
-// Delete the nfs-server pod. Only done once per KubeDescription().
-func nfsServerPodCleanup(c clientset.Interface, config VolumeTestConfig) {
```

Comment:
why move this func? Seem that any other provider that wants to run nfs to test their storage will want this func,